### PR TITLE
284 process tab

### PR DESCRIPTION
**Pull request by DutchBen**

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        tableStyle,
+        contentCellStyle,
+        headerCellStyle,
+        emptyCellStyle,
+        lastContentCellStyle,
+        lastHeaderCellStyle,
+    } = useWithOrchestratorTheme(getStyles);
+
+    return (
+        <div style={{ marginTop: 5 }}>
```
Comment (acidjunk) [inline packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx:37]: Css?

File: packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+            />
+        </EuiComment>
+    );
+};
+
+interface WfoProcessesTimelineProps {
+    subscriptionDetailProcesses: SubscriptionDetailProcess[];
+}
+
+export const WfoProcessesTimeline = ({
```
Comment (acidjunk) [inline packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx:121]: New style naming

File: packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+    subscriptionDetailProcesses,
+}: WfoProcessesTimelineProps) => {
+    return (
+        <>
+            <EuiSpacer size={'m'} />
+            {!subscriptionDetailProcesses && <WFOLoading />}
+            <EuiCommentList aria-label="Processes">
+                {subscriptionDetailProcesses && (
+                    <EuiCommentList aria-label="Processes">
```
Comment (acidjunk) [inline packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx:130]: Old style naming.

Probably better to stick to new for now; and do the rename in one single ticket.
(it will make the transformation easier)

File: packages/orchestrator-ui-components/src/messages/nl-NL.json [hunk 4]
```diff
@@ -183,7 +183,14 @@
             "created": "Aangemaakt",
             "endDate": "Eind datum",
             "note": "Notitie",
-            "noRelatedSubscriptions": "Geen gerelateerde subscriptions gevonden"
+            "noRelatedSubscriptions": "Geen gerelateerde subscriptions gevonden",
+            "hideTerminatedRelatedSubscriptions": "Hide terminated related subscriptions",
```
Comment (acidjunk) [inline packages/orchestrator-ui-components/src/messages/nl-NL.json:187]: I don't like EN text in the NL language file.

File: packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx (hunk 1 above)
Comment (ricardovdheijden) [reply packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx:37]: Indeed, css

File: packages/orchestrator-ui-components/src/components/WFOSubscription/styles.ts [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+export const getStyles = (theme: EuiThemeComputed) => {
+    const contentCellStyle = {
+        padding: theme.base,
+        borderBottom: theme.border.thin,
+    };
+
+    const headerCellStyle = css({
+        ...contentCellStyle,
+        paddingLeft: 0,
+        width: '250px',
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WFOSubscription/styles.ts:14]: I think it is better to use a multiple of base value, even when you cannot make it exactly 250. Base*16 would work too i assume?

File: packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+            </div>
+
+            <WfoProcessCard
+                subscriptionDetailProcess={subscriptionDetailProcess}
+            />
+        </EuiComment>
+    );
+};
+
+interface WfoProcessesTimelineProps {
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx:117]: Maybe move to its own file within this subscription folder

File: yarn.lock
Comment (ricardovdheijden) [inline yarn.lock:1]: Try to revert the changes in this whole file since you have effectively no changes in the package.json anymore

File: packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx (hunk 2 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx:132]: +1 right?

Comment (acidjunk) [reply packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx:132]: I was commenting about the inconsistency. But up to you if you do something with it: when we no do "new" code in the "new" style it will be, slightly, harder to do the big rename. I am ok with doing new components in the new style.

Just something to be aware of.

File: packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx (hunk 1 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx:38]: I can move the css but I worked with the assumption that we still do oneliners like this inline?

File: packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx (hunk 2 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx:132]: There is another ticket to do the bulk but I thought it was good to start using the new style. I'm not sure it makes it that much harder but I can pick up the ticket next so I have to deal with it

File: packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx (hunk 3 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx:141]: See previous

File: packages/orchestrator-ui-components/src/messages/nl-NL.json (hunk 4 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/messages/nl-NL.json:188]: Aangepast

File: packages/orchestrator-ui-components/src/components/WFOSubscription/styles.ts (hunk 5 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WFOSubscription/styles.ts:14]: updated

File: yarn.lock
Comment (DutchBen) [reply yarn.lock:1]: I've updated the yarn.lock and alls eems to work now

File: packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx (hunk 6 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx:128]: Well.. the processTimeline is in the right file but the other components perhaps

File: packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx (hunk 1 above)
Comment (DutchBen) [reply packages/orchestrator-ui-components/src/components/WFOSubscription/WfoProcessesTimeline.tsx]: Move the style to the table style and removed this one

**Review by acidjunk — APPROVED** [review]

**Review by ricardovdheijden — APPROVED** [review]
